docs/architecture/modern-web-apps-azure/choose-between-traditional-web-and-single-page-apps.md

```diff
@@ -86,8 +86,8 @@ Writing SPAs requires familiarity with JavaScript and/or TypeScript and client-s
 >   <https://angular.io>
 > - **React**
 >   <https://reactjs.org/>
-> - **Comparison of JavaScript Frameworks**  
->   <https://jsreport.io/the-ultimate-guide-to-javascript-frameworks/>
+> - **Vue.js**
+>   <https://vuejs.org/>
```

Comment:
The link for angular appears on a separate line. But for React and Vue, it appears on the same line. This is because Angular list item has 2 trailing whitespaces.

For consistency and better view:

- Either have all of them in the same line by removing the trailing spaces; or
- Have all of them in separate lines. But I won't use the backslash instead of two spaces for that.